music_assistant/helpers/tags.py

```diff
@@ -383,20 +383,40 @@ def lyrics(self) -> str | None:
 
     @property
     def track_loudness(self) -> float | None:
-        """Try to read/calculate the integrated loudness from the tags."""
-        if (tag := self.tags.get("r128trackgain")) is not None:
-            return -23 - float(int(tag.split(" ")[0]) / 256)
-        if (tag := self.tags.get("replaygaintrackgain")) is not None:
-            return -18 - float(tag.split(" ")[0])
+        """Try to read/calculate the integrated loudness from the tags (track level)."""
+        if tag := self.tags.get("r128trackgain"):
+            try:
+                value = int(tag.split(" ")[0]) / 256
+                return -23 - float(value)
```

Comment:
```suggestion
                gain_adjustment = int(tag.split(" ")[0]) / 256
                return -23 - gain_adjustment
```
Explicit conversion with `float()` is redundant since division always returns a float.